tests/unittests/topology/analyze_test.py

```diff
@@ -0,0 +1,224 @@
+# Copyright (C) 2017 UNINETT AS
+#
+# This file is part of Network Administration Visualized (NAV).
+#
+# NAV is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License version 2 as published by the Free
+# Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.  You should have received a copy of the GNU General Public
+# License along with NAV. If not, see <http://www.gnu.org/licenses/>.
+#
+
+import unittest
+
+import networkx as nx
+
+from nav.topology.analyze import AdjacencyReducer, Box, Port
+
+
+class AdjecencyReducerTestCase(unittest.TestCase):
```

Comment:
So the desired form for unittests are to use the assert-methods from unittest.TestCase (based on reviewing a lot more of the existing tests). Should I change to that even tough we do use py.test to run the tests and py.test's use of plain asserts are better in many ways? Or do we start using plain py.test tests?